app/src/main/kotlin/com/simplemobiletools/smsmessenger/receivers/MmsReceiver.kt

```diff
@@ -50,5 +51,5 @@ class MmsReceiver : MmsReceivedReceiver() {
         }
     }
 
-    override fun onError(context: Context, error: String) {}
+    override fun onError(context: Context, error: String) = context.toast(R.string.couldnt_download_mms)
```

Comment:
I think it's better to enable transitive R classes, they're gonna be the default from very near future
`android.nonTransitiveRClass=true`
saves you on build time tho

Reply:
Sure :)  Will do it in another PR soon as this one is optional.

> they're gonna be the default from very near future

I believe they are already default since AGP 8.0.0. I just didn't think the build speed impact was worth the readability so I disabled it 😁